xbmc/video/Teletext.cpp

```diff
@@ -1304,6 +1311,20 @@ void CTeletextDecoder::RenderPage()
   }
 }
 
+bool CTeletextDecoder::IsSubtitlePage(int pageNumber) const
+{
+  if (!m_txtCache)
+    return false;
+
+  for (const auto subPage : m_txtCache->SubtitlePages)
+  {
+    if (subPage.page == pageNumber)
+      return true;
+  }
```

Comment:
could be a nice use case for `std::any_of`

Reply:
I don't think it will work as the SubPages is a legacy array of structs, or at least couldn't find a way to use it without changing the array

Reply:
Ah, okay then.